examples_tests/39.DenoiserTonemapper/CommandLineHandler.hpp

```diff
@@ -41,19 +41,23 @@ Optional Parameters:
 -NORMAL_CHANNEL_NAME=normalChannelName
 -BLOOM_PSF_FILE=psfFilename
 
-NORMAL_FILE can only be specified if ALBEDO_FILE has been specified
-ALBEDO_CHANNEL_NAME can only be specified if ALBEDO_FILE has been specified
-NORMAL_CHANNEL_NAME can only be specified if NORMAL_FILE has been specified
-
 Note there mustn't be any space characters!
 All file's resolutions must match!
 Also you must not put another data like comments or behaviour will be undefined, the app'll crash (TODO: not crash on parse failure)
 
-Description and usage: (TODO: Update these)
+Description and usage: 
+
+COLOR_FILE: File containing color image data, it's required to run within.
+ALBEDO_FILE: File containing albedo image data, it is not required to run within.
+NORMAL_FILE: File containing normal image data, it is not required to run within, but it can only be specified if ALBEDO_FILE has been specified.
 
-~OPENEXR_FILE: OpenEXR file containing various channels~
+COLOR_CHANNEL_NAME: Channel name of an image for color image that the image will be assigned to. 
+ALBEDO_CHANNEL_NAME: Channel name of an image for albedo image that the image will be assigned to. It can only be specified if ALBEDO_FILE has been specified.
+NORMAL_CHANNEL_NAME: Channel name of an image for normal image that the image will be assigned to. It can only be specified if NORMAL_FILE has been specified.
```

Comment:
rephrase "Channel name for multilayered EXR images, a layer with most matching name will be chosen (one that contains the channel name as a substring at the earliest position)."

Reply:
done